linker/shared/src/main/scala/org/scalajs/linker/frontend/optimizer/OptimizerCore.scala

```diff
@@ -1425,9 +1428,9 @@ private[optimizer] abstract class OptimizerCore(
 
       case PreTransLocalDef(localDef @ LocalDef(tpe, _, replacement)) =>
```

Comment:
Consider dropping the destructuring of `LocalDef`. Or at least, replace `tpe` with `_` (it seems to be unused now).